src/components/form/ButtonConatiner.styled.js.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const ButtonContainer = styled.div`
+	button {
```

Comment:
Sorry, I didn't see this in the last code review. Why is this `div` your styled-component and not the `button` itself? I think it would be better if you created a styled-button instead of the styled-div, since the button inside your `ButtonContainer` is the only element with styling.

Reply:
P.S. your file name has some typos: double-`.js`and `Conatiner` 😉